WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -768,6 +769,7 @@ private void setupAddMenuPopup() {
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 adapter.notifyDataSetChanged();
 
+                AppLockManager.getInstance().setExtendedTimeout();
```

Comment:
Sorry for that! Forgot to set the PIN lock when I re-tested it, so it seemed to work for me 😁 . Addressed in dd92305